src/parser/utils.c

```diff
@@ -0,0 +1,51 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   utils.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: fratajcz <fratajcz@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2019/12/08 19:46:45 by fratajcz          #+#    #+#             */
+/*   Updated: 2020/04/10 00:08:46 by fratajcz         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "shell.h"
+
+int			mandatory_reserv_word(int expect_type)
```

Comment:
no consistency !!!!!

Reply:
changed for get_required_reserv_word(int expect_type)